src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -24,6 +24,140 @@ these dependencies for Axom.
     and experimentation?
   * Others?
 
+Determinism
+-----------
+
+We strive for as close to deterministic behavior in our builds as possible.
+By this, we mean that repeated builds should should act the same in the following
+regards:
+
+* Set of libraries with their options and versions
+* Compilers, compiler flags, and versions
+* Installed file and directory structure with permissions
+
+
+===========================================
+Build Scripts and Their Configuration Files
+===========================================
+
+There are three levels of build scripts or programs that drive TPL builds.
+As you move up the levels, and away from Spack, the scripts require less
+configuration and even build suites of builds at a time.
```

Comment:
> As you move up the levels, and away from Spack, the scripts require less
configuration and even build suites of builds at a time.

This end of this sentence could be clearer, but I'm struggling to find a better way to phrase it:
Maybe:
> ... and even _generate_ suites of builds at a time ?

Reply:
pushed a change that maybe clarifies it.. i agree its unclear.